Cargo.toml

```diff
@@ -14,11 +14,10 @@ repository = "https://github.com/eqrion/cbindgen/"
 travis-ci = { repository = "eqrion/cbindgen" }
 
 [dependencies]
-clap = "2"
+clap = { version = "2", optional = true }
 log = "0.4"
-serde = {version = "1.0", features = ["derive"]}
+serde = { version = "1.0", default-features = false, features = ["derive"]}
```

Comment:
Should this be part of this commit? Or should it be a different commit?

Reply:
`serde-derive` is redundant if you use the `derive` feature, and the default features aren't needed here. I can rebase it to a different commit.

Reply:
Rebased.